Sources/SwiftKuery/With/With.swift

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
We don't have subdirectories in Swift-Kuery, and WITH is too small to start one, please remove the subdirectory